src/artm_tests/repeatable_result_test.cc

```diff
@@ -80,6 +81,12 @@ TEST(RepeatableResult, RandomGenerator) {
   }
 }
 
+TEST(RepeatableResult, TokenHasher) {
```

Comment:
Remove this change (a similar change is done in https://github.com/bigartm/bigartm/pull/266)